projects/shared/assets/locale/messages.shared.de.xlf

```diff
@@ -0,0 +1,62 @@
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en-US" target-language="nl" datatype="plaintext" original="ng2.template">
+        <body>
+            <trans-unit id="shared.about.git-version" datatype="html">
+                <source>Git version</source>
+                <target>Git-Version</target>
+            </trans-unit>
+            <trans-unit id="shared.about.title" datatype="html">
+                <source>About Tailormap</source>
+                <target>Über Tailormap</target>
+            </trans-unit>
+            <trans-unit id="shared.about.unknown" datatype="html">
+                <source>Unknown</source>
+                <target>Unbekannt</target>
+            </trans-unit>
+            <trans-unit id="shared.about.version" datatype="html">
+                <source>Version</source>
+                <target>Version</target>
+            </trans-unit>
+            <trans-unit id="shared.color-picker.color-code" datatype="html">
+                <source>Color code</source>
+                <target>Farbcode</target>
+            </trans-unit>
+            <trans-unit id="shared.color-picker.color-required" datatype="html">
+                <source>Color is required</source>
+                <target>Farbe ist erforderlich</target>
+            </trans-unit>
+            <trans-unit id="shared.common.close" datatype="html">
+                <source>Close</source>
+                <target>Schließen</target>
+            </trans-unit>
+            <trans-unit id="shared.common.no" datatype="html">
+                <source>No</source>
+                <target>Nein</target>
+            </trans-unit>
+            <trans-unit id="shared.common.yes" datatype="html">
+                <source>Yes</source>
+                <target>Ja</target>
+            </trans-unit>
+            <trans-unit id="shared.language.dutch" datatype="html">
+                <source>Dutch</source>
+                <target>Niederländisch</target>
+            </trans-unit>
+            <trans-unit id="shared.language.english" datatype="html">
```

Comment:
I think we need a key for German/Duits/Deutsch in the code using these eg. language switch component otherwise this language will not show up. Also maybe a new `de` build
/Cc @geertplaisier

Reply:
I've added `de` in the language helper